src/expr/node_algorithm.h

```diff
@@ -91,6 +92,19 @@ void getSymbols(TNode n, std::unordered_set<Node, NodeHashFunction>& syms);
 void getSymbols(TNode n,
                 std::unordered_set<Node, NodeHashFunction>& syms,
                 std::unordered_set<TNode, TNodeHashFunction>& visited);
+
+/**
+ * For term n, this function collects the operators that occur in n.
+ * @param n The node under investigation
+ * @param syms The map (from each type to operators of that type) which the operators of
+ * n are added to
+ */
+void getOperatorsMap(TNode n, std::map<TypeNode, std::unordered_set<Node, NodeHashFunction>>& ops);
+/** Same as above, with a visited cache */
```

Comment:
It is better to write out the comment IMHO (in case we generate the documentation one day).

Reply:
The other option would be to explicitly refer to the name of the other function.